samples/python/bulk_edit_issues.py

```diff
@@ -13,9 +14,19 @@
 
 def bulk_edit_issue(issues):
     print ("Triggering Bulk Action for %d issues starting with id %s and ends with id %d" %((len(issues)), issues[0], issues[-1]))
-    api_params = "/?issue-ids=" + json.dumps(issues) + "&status=Resolved" #example
-    response = requests.put((ENDPOINT + api_params), auth=(API_KEY, ""))
+    api_params = {"issue-ids" : json.dumps(issues),
+                  "message-body" : "Your order will reach you shortly!", #example
+                  "status" : "Resolved"} #example
+    headers = {
+        'Authorization': 'Basic ' + base64.b64encode(API_KEY+':'),
```

Comment:
why `:` is added to the `API_KEY` here?

Reply:
No specific reason. Works without that as well. Wanted to retain username:password syntax.